packages/dropdowns/src/styled/field/StyledInput.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import styled, { css } from 'styled-components';
+import TextStyles from '@zendeskgarden/css-forms/dist/text.css';
+
+const hiddenStyling = css`
```

Comment:
Much of this styling appears superfluous for what you're trying to achieve.

Reply:
I've seen this reset used in a few libraries. [react-select](https://react-select.com/home) is one that comes to mind.

I've removed a few of the more extreme resets, but it should all be hidden by default.